packages/mobile/src/send/Send.tsx

```diff
@@ -357,6 +357,9 @@ const styles = StyleSheet.create({
   headerRightContainer: {
     paddingRight: 16,
   },
+  leftIcon: {
```

Comment:
This is not used anymore, so we can delete it

Reply:
thanks :) done!